src/ci/bin/test_papi.inc.sh

```diff
@@ -100,19 +100,7 @@ cromwell::private::papi::setup_papi_gcr() {
 
 cromwell::private::papi::setup_papi_service_account() {
     CROMWELL_BUILD_PAPI_AUTH_MODE="service-account"
-
-    # Delete all usages of CROMWELL_BUILD_PAPI_JSON_FILE from there if you delete this block!
-    if [[ "${CROMWELL_BUILD_TYPE}" == "centaurPapiV1" ]]; then
-        # Downgrade to an older service account on Papi V1 until refresh_token_no_auth_bucket.test is
-        # migrated/fixed/added for the Papi V2 credential below
-        CROMWELL_BUILD_PAPI_JSON_FILE="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/cromwell-service-account.json"
-    else
-        # This service account does not have billing permission, and therefore cannot be used for requester pays
-        CROMWELL_BUILD_PAPI_JSON_FILE="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/cromwell-centaur-service-account.json"
```

Comment:
Shouldn't this one be preserved?

Reply:
The variable is completely gone now and the literal value is used only [here](https://github.com/broadinstitute/cromwell/pull/6150/files/bf718d4dfcda91c76f07e0e4fe8cb614bbacc5ee#diff-342d95134a3c6c37cf687387f71731e7650273286f0e67fbca9c6cf6d081c64bR28)